src/shapes/Text/TextSVGExportMixin.spec.ts

```diff
@@ -41,48 +42,27 @@ describe('TextSvgExport', () => {
     });
 
     it('toSVG', () => {
-      const TEXT_SVG =
-        '<g transform="" style=""  >\n\t\t<text xml:space="preserve" font-family="Times New Roman" font-size="40" font-style="normal" font-weight="normal" style="stroke: none; stroke-width: 1; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: rgb(0,0,0); fill-rule: nonzero; opacity: 1; white-space: pre;" ><tspan x="-10" y="12.57" >x</tspan></text>\n</g>\n';
       const text = new FabricText('x');
-      expect(removeTranslate(text.toSVG())).toEqualSVG(
-        removeTranslate(TEXT_SVG),
-      );
+      expect(sanitizeSVG(text.toSVG())).toMatchSnapshot();
```

Comment:
we have `toEqualSVG` matcher
isn't it doing the same?
asking because it brings a bit inconsistency to the tests
that some are using it and some not

Reply:
toEqualSVG expect the expected svg to be passed in as a string.
I tried to write a toMatchSVGSnapshot but i got confused between chai and expect not sure why we are adding so many matchers there

Reply:
chai is what vitest uses for assertion library
expect is just the facade that is exposed to be extended, but chai is under the hood
and we used chai directly because via expect you can't extend the `toMatchSnapshot` matcher so it was the only way

anyway, I agree that we have too many matchers, maybe we could get rid of the toEqualSVG
I could have a look if you want

Reply:
done i guess.